build.sbt

```diff
@@ -28,8 +28,9 @@ lazy val tagless = (crossProject in file("freestyle-tagless"))
   .settings(name := "freestyle-tagless")
   .jsSettings(sharedJsSettings: _*)
   .crossDepSettings(commonDeps: _*)
-  .settings(resolvers += Resolver.bintrayRepo("kailuowang", "maven"))
+  .settings(scalametaSettings)
   .settings(
+    resolvers += Resolver.bintrayRepo("kailuowang", "maven"),
```

Comment:
Seems like newer versions are available in central and we may not need this additional repo declaration https://mvnrepository.com/artifact/com.kailuowang/mainecoon-core_2.12